english/chickens/chickens.md

```diff
@@ -0,0 +1,22 @@
+You are on a farm. Middle of Iowa.
+You notice you are wearing overhauls. A worn down house stands behind you.
+
+You remember hearing someone call your name.
+But you dont know whether to trust yourself.
+As you turn to look at the house, you realize something is at your feet.
+You recognize it as your pet chicken...
+
+You look up and suddenly, hearing the voice again.
+The voice says "Come on, what is the deal!"
+
+You look down, and realize it is the chicken speaking...
+
+Do you:
+
+[Kick it as far and as fast as possible](https://www.youtube.com/watch?v=6VuwxijKof0)
+
+[Realize that it's just a dream](../dream/dream.md)
+
+Respond 'Oh hey what is up, let's do our chores [Run towards the house](http://www.backyardchickens.com/t/579936/so-what-do-you-do-for-your-chicken-chores-d)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.